2003-02-15  Gwenole Beauchesne  <gbeauchesne@mandrakesoft.com>

	* ld/testsuite/ld-shared/main.c (main): -Wall fixes.
	
--- binutils-2.13.90.0.18/ld/testsuite/ld-shared/main.c.testsuite-Wall-fixes	2001-06-09 03:13:02.000000000 -0400
+++ binutils-2.13.90.0.18/ld/testsuite/ld-shared/main.c	2003-02-15 03:47:05.000000000 -0500
@@ -38,8 +38,6 @@ shlib_overriddencall2 ()
 int
 main ()
 {
-  int (*p) ();
-
   printf ("mainvar == %d\n", mainvar);
   printf ("overriddenvar == %d\n", overriddenvar);
   printf ("shlibvar1 == %d\n", shlibvar1);
@@ -62,6 +60,8 @@ main ()
   printf ("shlib_checkfunptr2 (main_called) == %d\n",
 	  shlib_checkfunptr2 (main_called));
 #endif
+ {
+  int (*p) ();
   p = shlib_getfunptr1 ();
   printf ("shlib_getfunptr1 () ");
   if (p == shlib_shlibvar1)
@@ -78,6 +78,7 @@ main ()
     printf ("!=");
   printf (" main_called\n");
 #endif
+ }
 #endif
   printf ("shlib_check () == %d\n", shlib_check ());
   return 0;
